websites/M/MEE6/presence.ts

```diff
@@ -2,59 +2,202 @@ const presence = new Presence({
 		clientId: "632002763483512843",
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
-let title: HTMLElement, search: HTMLInputElement;
-
+async function getStrings() {
+	return presence.getStrings(
+		{
+			browse: "general.browsing",
+			search: "general.searchFor",
+			viewCategory: "general.viewCategory",
+			buttonViewPage: "general.buttonViewPage",
+			viewHome: "general.viewHome",
+		},
+		await presence.getSetting<string>("lang").catch(() => "en")
+	);
+}
+function capitalizeFirstLetter(string: string) {
+	return (
+		string.charAt(0).toUpperCase().trim() + string.slice(1).toLowerCase().trim()
```

Comment:
You should probably trim the string first, cause rn you are getting the character at position 0, which would be 1 character and then trimming it, which does nothing...

Reply:
oh good point